src/com/session/PostServlet.java

```diff
@@ -0,0 +1,76 @@
+package com.session;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.LinkedHashMap;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.codehaus.jackson.map.ObjectMapper;
+
+import com.google.appengine.labs.repackaged.org.json.JSONException;
+
+import PojoClasses.PostService;
+
+//import org.mortbay.util.ajax.JSON;
+
+public class PostServlet extends HttpServlet {
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = 1L;
+	LinkedHashMap<String, Object> finalJson = new LinkedHashMap<String, Object>();
+
+	public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+		PostService ps = new PostService();
+		resp.setContentType("application/json");
+		PrintWriter out = resp.getWriter();
+		ObjectMapper mapper = new ObjectMapper();
+		
+		
+
+		PostPojo post = null;
+
+		try {
+			String path = req.getPathInfo().substring(1);
+			int requestNumber = Integer.parseInt(path);
```

Comment:
instead of doing exception, you already know the requestNumber can be null, so can you simply do a if condition check